include/exadg/convection_diffusion/throughput.h

```diff
@@ -29,9 +29,9 @@
 
 // deal.II
 #include <deal.II/base/exceptions.h>
-#include <deal.II/base/parameter_handler.h>
```

Comment:
Is this removal mandatory?

Reply:
No, see [c3f0a7d](https://github.com/exadg/exadg/pull/382/commits/c3f0a7daf7857d39dcd9474eb86602d5bde96c1f)